pkg/proc/breakpoints.go

```diff
@@ -195,6 +216,15 @@ func (bp *Breakpoint) CheckCondition(thread Thread) BreakpointState {
 	return bpstate
 }
 
+// checkHitCond evaluates bp's hit condition on thread.
+func (bp *Breakpoint) checkHitCond(bpstate BreakpointState, thread Thread) BreakpointState {
+	if bp.HitCond != nil {
```

Comment:
Either move this if to the caller or move all of the ifs around the call here. If you move them here invert the condition and return early.

Reply:
Done